lib/src/facts/linux/virtualization_resolver.cc

```diff
@@ -117,6 +117,11 @@ namespace facter { namespace facts { namespace linux {
                 return true;
             }
             // Take the first line that isn't an error/warning
+            // unless it's "xen", in which case we expect a second
+            // line with more useful information
```

Comment:
Not as far as I can tell. Running as a nonpriviliged user spits out "xen0" but not "xen" on the hypervisor, so I think we're safe here.

I could have it latch 'xen' into the value field to be on the safe side. Then if somehow we don't get the detailed line we'll still have *an* answer